lib/ui/src/modules/ui/components/stories_panel/index.js

```diff
@@ -1,19 +1,22 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import pick from 'lodash.pick';
-import Header from '../header';
+import glamorous from 'glamorous';
+
+import { Header } from '@storybook/components';
 import Stories from './stories_tree';
 import TextFilter from './text_filter';
 
-const scrollStyle = {
-  height: 'calc(100vh - 105px)',
-  marginTop: 10,
```

Comment:
Looks like you've lost this margin

Reply:
No just moved it to an other wrapper, or did you find something mis-aligned?

Reply:
Fixed